docs/_includes/header.html

```diff
@@ -38,8 +38,63 @@
 
         <div class="s-topbar--searchbar w100 wmx3 sm:wmx-initial js-search">
             <div class="s-topbar--searchbar--input-group">
-                <input id="searchbox" type="text" placeholder="Search Stacks…" value="" autocomplete="off" class="s-input s-input__search" />
+                <!-- TODO remove added attributes before merging topbar popover -->
```

Comment:
Good suggestion! I couldn't find a simple way to use Stacks classes for that popover (I can't even [update the config](https://stackexchange.slack.com/archives/C27RWNQN9/p1611196118000600?thread_ts=1611184636.095000&cid=C27RWNQN9) without going through Algolia), so I took the easy way out in https://github.com/StackExchange/Stacks/pull/1739/commits/31d7055dcb54e0644dad9c9318499e6428be6661 and just added a couple of styles lifted from the topbar.

How do you feel about using those custom properties outside of the topbar? I was on the fence between that or duplicating the custom properties in the context of the custom algolia popover CSS.

Reply:
Since those are only applied to the docs I think it's ok. I think we need to make sure though that somebody doesn't get tempted to do that elsewhere. Duplicating with a comment is also a viable option. Perhaps a comment should be added anyway saying "don't do this, we are only doing this because we cannot apply our classes easily to the algolia popover".